rdr_service/model/bq_participant_summary.py

```diff
@@ -257,6 +257,13 @@ class BQParticipantSummarySchema(BQSchema):
 
     test_participant = BQField('test_participant', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.NULLABLE)
 
+    suspension_status = BQField('suspension_status', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
+    suspension_status_id = BQField('suspension_status_id', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.NULLABLE)
+    suspension_time = BQField('suspension_time', BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
```

Comment:
These are duplicate of previous suspension fields, see line: 212.

Reply:
Thanks.  Change to remove duplicate lines committed